backend/local_deployment/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.2'
+
+services:
+  db:
+    build: ./postgres
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: postgres
```

Comment:
we should put this in an environment file and create a environment.template file. This way, chances are lower to forget changing this for production.

Reply:
This docker-compose is explicitly only for the local setup. The productive deployment will be with kubernetes